sdk/attestation/azure-security-attestation/azure/security/attestation/_models.py

```diff
@@ -948,3 +948,13 @@ def __init__(self, message):
         super(  # pylint: disable=super-with-arguments
             AttestationTokenValidationException, self
         ).__init__(self.message)
+
+class TpmAttestationResult(object):
+    """Represents the Tpm Attestation response data returned from the attestation service
+    as a result of a call to :meth:`azure.security.attestation.AttestationClient.attest_tpm`.
+
+    :keyword bytes result: The result of the operation.
+    """
+
+    def __init__(self, result: bytes):
+        self.result = result
```

Comment:
I would name this attribute `data` intead of `result`. We wouldn't normally use `result` as an attribute name, and it's already present in the model name :)